packages/ui/src/__mocks__/logto.tsx

```diff
@@ -0,0 +1,2 @@
+export const AppLogo = 'https://avatars.githubusercontent.com/u/88327661?s=200&v=4';
+export const AppContext = 'Build user identity in a modern way';
```

Comment:
should be `content` instead of `context`. but `content` is still confusing, how about `app headline` / `app slogan`

Reply:
copied from Figma~ could update